### ISSUE TITLE
Update Ocean2D example in FabUQCampaign.py to use the refactored EasyVVUQ

### ISSUE THREAD

**Issue by djgroen** [djgroen/FabSim3#100]
It's this function:
`def ocean2D(**args):`
in this file:
http://www.github.com/wedeling/FabUQCampaign.py

SOFT DEADLINE: 23rd of May
HARD DEADLINE: 29th of May (but then you'll also have to revise the tutorial document in FabUQCampaign! ;))

**Comment by wedeling** [comment]
I'll have a look at this early next week

**Comment by wedeling** [comment]
Hi Derek, quick question. How do I run your ocean2D subroutine? My old script doesn't work any more now that run_UQ_sample is removed.

**Comment by djgroen** [comment]
Hi Wouter,

That's strange, I thought that I'd kept your old script intact... Perhaps
it's best to return run_UQ_sample to its original location then, so that we
can keep using the old script as well?

The new script is not fully functional yet, because I have to wait for
EasyVVUQ to be updated.
It can be run like your old script though (if run from config/Ocean2D), but
I just noticed that I forgot to commit the new config files to the
repository from my laptop :/. I'll try to get that sorted out tomorrow
morning :).

Cheers & apologies for the mistake there!
-Derek

**Comment by djgroen** [comment]
Okay, I committed the fix now. You can run the script through FabSim3 using

```
cd config_files/ocean2d
fab localhost ocean2d:ocean2d:label=1
```

I hope to be able to remove the first line after the refactoring with EasyVVUQ is done.

**Comment by wedeling** [comment]
Hi Derek, I have Stochastic Collocation working with the refactored EasyVVUQ, although I haven't pushed the new version yet. Anyway, when I run your command above I get a `Command(s) not found:ocean2d` error, and the same when I try to run `uq_ensemble` from within my EasyVVUQ script. It seems I cannot access any subroutine in `FabUQCampaign.py`. Is there something which I might be messing up?

**Comment by wedeling** [comment]
I've pushed both the refactored advection diffusion and ocean model examples. I also updated both tutorials (the readme and the separate ocean tutorial). I'll fine tune them once we work out the issue above.

**Comment by wedeling** [comment]
Hi @djgroen 

I have managed to fix the issues above with the help of Hamid. Turns out that the import statements above the `ocean_2d` task in `FabUQCampaign,py` were creating a problem. If I comment them out, FabSim recognizes the other subroutines in the same file again. 

I had another issues as well, which I fixed and pushed:

1) If I keep `sim_ID` fixed (i.e. the `sim_ID` needed for `campaign2ensemble`), and rerun a campaign with a different polynomial order, the old samples get executed again, along side the new ones. This is because the old samples do not get deleted from `<fab_home>/config_files/<sim_ID>/SWEEP`. I fixed this by assigning a dynamic `sim_ID` (the same name used for the EasyVVUQ campaign), such that it is different every time a campaign is launched.

2) The previous version had quite a lot of hard coded variables. I now made a more generic `run_FabUQ_ensemble` subroutine to run the ensemble. It is located in the same file as the example scripts. Only the fab home directory needs to be hard coded now.

I've updated the advection diffusion tutorial (https://github.com/wedeling/FabUQCampaign/blob/master/README.md) and the ocean tutorial (https://github.com/wedeling/FabUQCampaign/blob/master/Tutorial_ocean.md)
